tests/main/snap-run-explain/expected

```diff
@@ -0,0 +1,72 @@
+The extra output from snap run --explain is not suitable for parsing.
+
+[snap run]
+
+Invoked as "snap"
+Inferred application details:
+  snap name: test-snapd-sh
+  app name: test-snapd-sh
+  confinement: strict
+  base: core (implicit)
+Will exec through the following tools:
+  - snap-confine (execution environment and sandboxing)
+  - snap-exec (command chain and environment variables)
+Creating user data directories
+  - ~/snap/test-snapd-sh/x1
+  - ~/snap/test-snapd-sh/common
+Creating symbolic link "x1" -> "/root/snap/test-snapd-sh/current"
+Executing snap-confine /snap/core/8250/usr/lib/snapd/snap-confine
+  with arguments: snap.test-snapd-sh.test-snapd-sh /usr/lib/snapd/snap-exec test-snapd-sh -c /bin/true
+  with altered environment
+    HOME: /root/snap/test-snapd-sh/x1
+    SNAP: /snap/test-snapd-sh/x1
+    SNAP_ARCH: amd64
+    SNAP_COMMON: /var/snap/test-snapd-sh/common
+    SNAP_DATA: /var/snap/test-snapd-sh/x1
+    SNAP_INSTANCE_NAME: test-snapd-sh
+    SNAP_LIBRARY_PATH: /var/lib/snapd/lib/gl:/var/lib/snapd/lib/gl32:/var/lib/snapd/void
+    SNAP_NAME: test-snapd-sh
+    SNAP_REVISION: x1
+    SNAP_USER_COMMON: /root/snap/test-snapd-sh/common
+    SNAP_USER_DATA: /root/snap/test-snapd-sh/x1
+    SNAP_VERSION: 1.0
+    XDG_RUNTIME_DIR: /run/user/0/snap.test-snapd-sh
+
+[snap-confine]
+
+Execution environment:
+  - Root file system is /snap/core/current
+  - Creating new per-snap mount namespace
+    desired fstab /var/lib/snapd/mount/snap.test-snapd-sh.fstab
+    effective fstab /run/snapd/ns/snap.test-snapd-sh.fstab
+    info file /run/snapd/ns/snap.test-snapd-sh.info
+  - Using private /tmp directory
+  - Apparmor profile: snap-update-ns.test-snapd-sh
+    source: /var/lib/snapd/apparmor/profiles/snap-update-ns.test-snapd-sh
+    binary: (loaded into the kernel, cached by the platform)
+  - Apparmor not compiled into snap-confine
+
+[snap-update-ns]
```

Comment:
here things get confusing, where does snap-update-ns output finishes ?  maybe

<< snap-update ns (description) >>
<< ... snap-confine >>  once we are back?

Reply:
another approach is to consider snap-update-ns really an implementation detail of snap-confine and not prefixing it at all

Reply:
I think it deserves its own section simply because there are sections for snap run / confine / exec (we could reconsider but I think explain was about not hiding it). I agree that the output is a bit intermixed and it needs improved termination.